src/laboratory/forms.py

```diff
@@ -863,3 +863,29 @@ class Meta:
             'name': genwidgets.TextInput,
             'description': genwidgets.Textarea
         }
+
+
+class MaterialForm(MaterialCapacityObjectForm, forms.ModelForm):
+
+    def __init__(self, *args, **kwargs):
+        modal_id = kwargs.pop('modal_id')
+        laboratory_pk = kwargs.pop('laboratory_pk')
+        super(MaterialForm, self).__init__(*args, **kwargs)
+        self.fields['object'].required = False
+        self.fields['is_container'].initial = True
```

Comment:
This line can be removed self.fields['is_container'].initial = True  and you can define this initial from create_from in view_material_list